scripts/battlefields/Temenos/temenos_eastern_tower.lua

```diff
@@ -279,12 +279,15 @@ content.groups =
             "Dark_Elemental_E",
         },
 
-        -- NOTE: Elementals take double physical damage because their family resistance is 25% so it totals to 50% resistance
+        -- NOTE: Elementals in here take 50% physical damage instead of the usual 25%
         mods =
         {
-            [xi.mod.UDMGPHYS ] = 10000,
-            [xi.mod.UDMGMAGIC] = -5000,
-            [xi.mod.FASTCAST ] = 20,
+            [xi.mod.SLASH_SDT   ] = 500,
+            [xi.mod.PIERCE_SDT  ] = 500,
+            [xi.mod.IMPACT_SDT  ] = 500,
+            [xi.mod.HTH_SDT     ] = 500,
+            [xi.mobMod.DETECTION] = xi.detects.HEARING,
```

Comment:
Detection change and fast cast change weren't mentioned in the commit or the pr description. If accurate to retail, they should have been mentioned.

Reply:
I can't test on retail but ffxiclopedia says that they only detect sound in East.
https://ffxiclopedia.fandom.com/wiki/Fire_Elemental

Reply:
Never mind, before this PR only 2nd Floor Elementals detect sound, just let me know how to go about this.

Reply:
Next time just add it to your commit message. We can add it in if we squash your commits, but I don't like editing people messages.

Reply:
> Next time just add it to your commit message. We can add it in if we squash your commits, but I don't like editing people messages.

I thought this was pretty clear. I was **not** looking for another commit removing it after a commit adding it.